src/content/overview/fleet-management/cluster-management/installation-name/index.md

```diff
@@ -0,0 +1,44 @@
+---
+linkTitle: Installation names
+title: Picking an installation name
+description: Every Giant Swarm installation has a unique name. Learn the rules to select a proper name for a new installation.
+weight: 100
+menu:
+  principal:
+    parent: overview-fleet-management-clusters
+    identifier: overview-fleet-management-clusters-installation-name
+user_questions:
+  - How do I determine the name for my Giant Swarm installation / management cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+last_review_date: 2024-11-26
+---
+
+A Giant Swarm installation is the management cluster where all our platform components run. It's attached to a cloud provider account and region, or an on-premise data center, and it's the central point and API to use the platform capabilities.
+
+## Why an installation name
+
+Every Giant Swarm installation has an unique name. This name is part of the DNS zone used for all endpoints of the central platform.
+
+When setting up a new installation for you, a name is needed. Think of this name as your handy alias for everything you do with Giant Swarm in a specific infrastructure provider region, for example _Google Cloud Finland (europe-north1)_.
```

Comment:
Technically, couldn't you use the same installation to manage WCs on multiple regions?